autoprotocol/protocol.py

```diff
@@ -1035,102 +1046,278 @@ def liquid_handle_dispense(
                 volume=[Unit(_, "uL") for _ in range(1, 13)]
             )
 
+        Using multiple chips in a single source tube. The following example will
+        dispense 100:microliters to the first 3 columns of the destination plate.
```

Comment:
I am still confused why the number of chips would have an impact on the destination wells.

Shouldn't that be totally independent? If we want to dispense to a single column and we request 3 chips, it shouldn't matter doesn't it? I think that become to linked to the device function.

Reply:
we're definitely on the same page, the docstring example just needs to be updated to reflect the code.

the new example should say `destination = [destionation.wells_from(0,3)]`

Reply:
I am still confused because now we specify 3 at two different places:

```
            intake_hoses = 3
            source: List[Tuple[Well, int]] = [(source, intake_hoses)]
            destination = [destination.wells_from(0, 3)]
```

What does the first 3 mean? and what the second 3 means?

Is the second 3 the number of column we want to dispense too? If yes can we add a variable name to make it more clear!

Reply:
the first 3 is the number of chips that should be fed into the source tube. the second 3 is an argument to the `.wells_from()` method which will return a wellgroup of `destination.well(0)`, `destination.well(1)` and `destination.well(2)`

the reason it's included in this example is because it makes more sense to have 3 chips dispense to 3 columns... but it could be any number

Reply:
yeah, i'll make it more clear as well though

Reply:
OK, that's what I guessed after, we could use 6 for the second to show it doesn't have to be equal!

Reply:
i changed it to 5... same thing though haha

Reply:
haha thanks, I think it is more clear now, and it makes sense